README.md

```diff
@@ -23,15 +23,71 @@ Mental health problems are something that everyone struggles with at various poi
 
 MindGPT sets out to increase ease of access to this information.
 
-# &#128064; Follow along!
+# &#128064; Follow along
 
 This project is in active development at Fuzzy Labs and you can follow along!
 
 The repository for this project is one method where you can monitor progress - we're always raising pull requests to move the project forward. Another way you can do this is to follow our blog. We'll be posting periodic updates as we tick off sections of the project. If you're new to the project, the best place to start is our [project introduction](https://www.fuzzylabs.ai/blog-post/mindgpt-an-introduction).
 
 # &#127939; How do I get started?
 
-We're working on this! Coming soon...
+## Deployment pipeline
+
+To run the zenml deployment pipeline, the resources required by seldon have to provisioned. `matcha` tool can help you in provisioning these resources.
+
+Provision resources using `matcha`
+
+```bash
+matcha provision
+```
+
+Assign output for resources provisioned by `matcha` to variables.
+
+```bash
+function get_state_value() {
```

Comment:
I'd probably put this into a bash script, e.g., `setup.sh` with any prerequisite steps added to the readme before telling the user to run the setup script